src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -236,12 +235,13 @@ class WorkspaceReimburseView extends React.Component {
                                     <TextInput
                                         label={this.props.translate('workspace.reimburse.trackDistanceRate')}
                                         placeholder={this.state.outputCurrency}
-                                        onChangeText={value => this.setRate(value)}
+                                        onChangeText={value => this.setRate(value.replace(/[^0-9.]/g, ''))}
```

Comment:
I think we should move this logic inside `setRate` or let's create a new method where we will replace the non numeric chars and call `setRate` inside it with the new value?